app/views/competitions/_nav.html.erb

```diff
@@ -43,7 +43,7 @@
               { text: t('.menu.orga_view'), path: edit_competition_path(@competition), icon: "lock" },
               { text: t('.menu.event_view'), path: edit_events_path(@competition), icon: "cubes" },
               { text: t('.menu.schedule_view'), path: edit_schedule_path(@competition), icon: "calendar" },
-              { text: t('.menu.payment_view'), path: competitions_payment_setup_path(@competition), icon: "cc stripe" },
+              { text: t('.menu.payment_view'), path: competition_payment_integration_setup_path(@competition), icon: "cc stripe" },
```

Comment:
Do we want to take this opportunity to change the icon away from the stripe icon? (I have it on my backlog checklist so not a nit we're in danger of losing)

Reply:
sure! Changed it to a money bill, which is more intuitive and less tightly coupled to Stripe